triggers/simulations/overview.rst

```diff
@@ -0,0 +1,461 @@
+.. Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
This file will disappear when https://github.com/digital-asset/daml/pull/16903 is merged